types.go

```diff
@@ -43,3 +43,11 @@ type IPRanges struct {
 	RangeEnd   string
 	Gateway    string
 }
+
+// BandWidth defines the ingress/egress rate and burst limits
+type BandWidth struct {
+	IngressRate  int
```

Comment:
int64 ?

Reply:
Actually I would prefer `uint64` because setting a limitation for a negative value is pretty senseless (at least from my point of view).

Reply:
sure that works!

Reply:
Changed it, thanks for your comment!